style.css

```diff
@@ -0,0 +1,84 @@
+/* @media all and (max-width: 500px) { */
+* {
+  font-family: sans-serif;
+  padding-left: 1%;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+}
```

Comment:
- [ ] Kindly implement the requirement suggested by the previous reviewer. In the image attached below, you can see the white patch that is circled. Kindly adjust it and make it look like the Figma template you chose.

### Your Design

![Untitled](https://user-images.githubusercontent.com/77106962/205169654-4a0aa3cd-c90a-421a-bf6b-1e58b4eed34c.png)


### Figma Design

![image](https://user-images.githubusercontent.com/77106962/205170286-884709ed-8638-4739-82c5-0dad85f7651c.png)